### ISSUE TITLE
Problem: adding hashpower and energy consumption for hardware devices is not bountified

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#247]
Problem: adding hardware at /hashpower is not bountified. Early on, this probably has to be incentivised. However maybe a screenshot of the user's mining rig or something is needed for verification. Alternatively, maybe outliers for each hardware device can be purged, or perhaps some kind of flagging or something.

**Comment by anbud** [comment]
@gazhayes, I agree, adding a bounty to add hash power data would be a nice thing to speed up data collection. A time based bounty that we use in other places is perfectly fine here.
As for verification, users could easily pull images of mining rigs from the internet, I believe that some flagging system would be sufficient here. We could also algorithmically check if the data is valid. We currently calculate the average, so if the data differs too much from the average, something is probably wrong, and we can flag it.
Nonetheless, adding an image of the mining rig could be nice thing for the UI.

**Comment by gazhayes** [comment]
> users could easily pull images of mining rigs from the internet

What I did with wallet images etc to try and combat this is MD5 the image and not allow any more uploads of that image. Of course they could just slightly change the image, but it forces people to be a bit more determined to cheat.

> I believe that some flagging system would be sufficient here

Yes, it works with wikipedia, will probably work here.

**Comment by anbud** [comment]
We could do that with images, you're right, it would probably force them to give up on trying to cheat the system. 
Maybe we could do it like this, let them upload the image, check the checksum and if everything is ok, let them add the data. (No separate image verification by moderator as when adding wallet images)
If other users notice that something is not right, they can just flag the data, and the moderator can take action then.
So, basically, a combination of proof and a flagging system would be quite sufficient here.

**Comment by anbud** [comment]
Done in #263.
